src/main/java/software/aws/neptune/gremlin/sql/SqlGremlinConnection.java

```diff
@@ -53,6 +53,6 @@ public QueryExecutor getQueryExecutor() throws SQLException {
 
     @Override
     public String getDriverName() {
-        return "neptune:sqlgremlin";
```

Comment:
Not sure about these getDriverName updates. They currently match our connection string setup:
`jdbc:<getDriverName>://<whatever>
`and _I think_ that lowercase is standard for this, but I am not sure.

@kylepbit Can you weight in on JDBC naming conventions here?

Reply:
Driver name is not related to the driver URL, you can put whatever you want here such as "JDBC Driver for AWS Neptune".

Reply:
In that case I would suggest 
"JDBC Driver for <language variant> Amazon Neptune"
Trying to avoid branding as AWS directly because of the Chinese trademark thing. Open to other things if you have something that you think sounds better though, Simon.

Reply:
I would suggest "Amazon Neptune JDBC" to align with other drivers.